source/Calamari/Kubernetes/Conventions/Helm/HelmManifestAndStatusReporter.cs

```diff
@@ -108,9 +120,15 @@ async Task<string> PollForManifest(RunningDeployment deployment,
                     await Task.Delay(TimeSpan.FromSeconds(1), ct.Token);
                 }
 
-            if (string.IsNullOrWhiteSpace(manifest))
+            //it's possible that some manifests doesn't
+            if (string.IsNullOrWhiteSpace(manifest) && !didSuccessfullyExecuteCliCall)
```

Comment:
```suggestion
            if (!didSuccessfullyExecuteCliCall)
```
If the cli call failed for some reason, this should probably throw regardless of the cli output.